handover_api/views.py

```diff
@@ -36,8 +36,9 @@ def send(self, request, pk=None):
             raise AlreadyNotifiedException(detail='Handover already in progress')
         accept_path = reverse('ownership-prompt') + "?token=" + str(handover.token)
         accept_url = request.build_absolute_uri(accept_path)
-        send_handover(handover, accept_url)
-        handover.mark_notified()
+        message = HandoverMessage(handover, accept_url)
+        message.send()
+        handover.mark_notified(message.mime_text)
```

Comment:
I don't like how this value is changing names from mime_text to email_text.
I initially thought you were storing MIME type text ('plain/text') in the database for some reason. Message.mime_text explains what is going on, but I would prefer having it consistently named email_text or email_content.